### PR TITLE
[SDESK-2798] Add endpoint to support auto suggest for contact organis…

### PR DESCRIPTION
**Pull request by marwoodandrew**
…ation names

### CODE REVIEW

**Comment by mdhaman** [comment]
remove `'repo': 'contacts'` from the query string otherwise it looks fine.

File: apps/contacts/service.py [hunk 1]
```diff
@@ -48,3 +49,29 @@ def on_deleted(self, doc):
...
+    def get(self, req, lookup):
+        """
+        Search for organisation matching the passed q parameter
+        :param req:
+        :param lookup:
+        :return: List of matching organisation strings
+        """
+        new_req = ParsedRequest()
+
+        q_str = 'organisation:' + '* organisation:'.join(req.args.get('q', '').split()) + '*'
```
Comment (petrjasek) [inline apps/contacts/service.py]: is it supposed to be `organisation:* organisation:<q>*` ? I think [match_phrase_query](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-match-query.html#query-dsl-match-query-phrase-prefix) could be better for the prefix search

Comment (marwoodandrew) [reply apps/contacts/service.py]: The idea is to wildcard each term. Not strictly a phrase search as I'm trying to match say "insurance australia" or "australian insurance". Each term is prefixed with "organisation:" An alternative might have been to set the default field to 'organisation'

Comment (petrjasek) [reply apps/contacts/service.py]: I see, makes sense then

**Comment by mdhaman** [comment]
@petrjasek don't merge this PR. We will merge it in the morning. @sjunaid was having issues with `endpoint name`.

**Comment by marwoodandrew** [comment]
Changed the title of the end point.

**Pull request merged**